src/components/Team/Team.css

```diff
@@ -0,0 +1,39 @@
+.section-team{
+    margin: 0 auto;
+    max-width: none;
+    position: relative;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    box-shadow: inset 0 200px 200px -200px #fff, inset 0 -200px 200px -200px #fff;
+    display: -ms-flexbox;
+    display: -webkit-box;
+    display: flex;    
+    padding: 50px 90px 140px 90px;
+    -ms-flex-pack: justify;
+    -webkit-box-orient: horizontal;
+    -webkit-box-direction: normal;
+    -ms-flex-flow: row wrap;
+    flex-flow: row wrap;
+    -webkit-box-pack: justify;
+    justify-content: space-between;
+    border-bottom: 1px solid #dfdfdf;
+}
+.section-container-team {
+    margin: 0 auto;
+    max-width: 1120px;
+    padding: 90px 20px 50px;
+    position: relative;
+    text-align: center;
+}
+.team-container{
+    display: flex;
+    flex-wrap: wrap;
+}
+.team-card{
+    margin: 10px 10px 0 0;
+    max-width: 280px;
```

Comment:
@isuruAb I tried doing that but the cards were not aligned properly for phones. Please check the changes now!